<?xml version="1.0"?>
<launch>

  <!-- Starts the hardware on the real robot -->
  <arg name="robot_name" default="nono_jetson" />
  <arg name="hardware_interface_dt" default="0.02" />

  <node name="$(arg robot_name)_robot_hardware" output="screen"
        pkg="nono_jetson" type="nono_jetson_hardware_interface">
    <param name="hardware_interface_dt" type="double" value="$(arg hardware_interface_dt)" />
    <remap from="/joint_states" to="/$(arg robot_name)/joint_states" />
  </node>

</launch>

